src/Core/Prefab/GlobeView.js

```diff
@@ -186,12 +248,21 @@ function GlobeView(viewerDiv, coordCarto) {
     this.wgs84TileLayer = wgs84TileLayer;
 
     this.mainLoop.addEventListener('command-queue-empty', () => {
-        viewerDiv.dispatchEvent(new CustomEvent('globe-built'));
+        this.dispatchEvent({ type: 'globe-built' });
     });
 
+    const fn = () => {
+        this.removeEventListener('globe-built', fn);
+        this.dispatchEvent({ type: 'initialized' });
+    };
+
+    this.addEventListener('globe-built', fn);
```

Comment:
This looks like a convoluted way to emit `initialized` on the first time `globe-built` is being fired.
I guess you could get rid of `globe-built` completely.
(and use `GLOBE_INITIALIZED` instead of `initialized`)